README.md

```diff
@@ -282,6 +282,39 @@ Lastly we have higher level libraries in `/lib`. These are javascript specific l
 1. `/clients`. This sub directory is code generated and depends on code published from `/packages` . It is 1:1 with AWS services and operations. Manual edits should generally not occur here. These are published to NPM under `@aws-sdk/client-XXXX`.
 1. `/lib`. This sub directory depends on generated code published from `/clients`. It wraps existing AWS services and operations to make them easier to work with in Javascript. These are published to NPM under `@aws-sdk/lib-XXXX`
 
+### Streams
+
+Certain command outputs include streams, which have different implementations in NodeJS and browsers. For convenience, a set of stream handling methods will be merged (`Object.assign`) to the output stream object, as defined in [SdkStreamMixin](https://github.com/aws/aws-sdk-js-v3/blob/ad99b66944b6fe1ce83082c1a33193dff033cfaf/packages/types/src/serde.ts#L77-L81).
```

Comment:
Although it's not done in rest of the document, I would advise using new lines after 80-120 manually characters. It helps in reading non-rendered version.

Also, in internal docs I've been adding [links at the bottom](https://commonmark.org/help/tutorial/07-links.html). We can start doing it with this PR may be?

Ditto on line 289.